### PR TITLE
use sam for groups

### PR DESCRIPTION
**Pull request by zarsky-broad**

### CODE REVIEW

**Review by MattPutnam — APPROVED** [review]

File: src/pages/groups/Group.js [hunk 1]
```diff
@@ -186,11 +186,11 @@ export class GroupDetails extends Component {
...
     try {
       this.setState({ loading: true, creatingNewUser: false, editingUser: false, deletingUser: false, updating: false })
-      const { membersEmails, adminsEmails } = await Rawls.group(groupName).listMembers()
+
       this.setState({
         members: _.sortBy('email', _.concat(
-          _.map(adm => ({ email: adm, role: 'admin' }), adminsEmails),
-          _.map(mem => ({ email: mem, role: 'member' }), membersEmails)
+          _.map(adm => ({ email: adm, role: 'admin' }), await Sam.group(groupName).listAdmins()),
+          _.map(mem => ({ email: mem, role: 'member' }), await Sam.group(groupName).listMembers())
```
Comment (panentheos) [inline src/pages/groups/Group.js]: nit: These could be parallelized

File: src/pages/groups/Group.js [hunk 2]
```diff
@@ -74,7 +74,7 @@ class NewUserModal extends Component {
 
     try {
       this.setState({ submitting: true })
-      await Rawls.group(groupName).addMember(role, userEmail)
+      await Sam.group(groupName).addMember(role, userEmail)
```
Comment (panentheos) [inline src/pages/groups/Group.js:77]: TOL: Makes me wonder if we should be abstracting over the service locations in our ajax code. There's no real reason that the code at this level needs to know what particular server it's hitting.

**Review by panentheos — APPROVED** [review]
Looks good.

File: src/pages/groups/Group.js (hunk 2 above)
Comment (zarsky-broad) [reply src/pages/groups/Group.js:77]: I've thought about that before, I think it's a good idea. I'll do another PR after this.

**Pull request merged**